.github/workflows/build-and-test-arm.yaml

```
@@ -6,7 +6,15 @@ on:
   workflow_dispatch:
 
 jobs:
+  pre_condition:
```

Or I think this is good.

```suggestion
  prevent-running-on-forks:
```

---

Sorry, I overlooked this comment. 1f1451f6